src/main/views/MattermostWebContentsView.ts

```diff
@@ -426,6 +428,14 @@ export class MattermostWebContentsView extends EventEmitter {
 
     private loadSuccess = (loadURL: string) => {
         return () => {
+            const serverInfo = ServerManager.getRemoteInfo(this.view.server.id);
+            if (serverInfo?.serverVersion && semver.lt(serverInfo.serverVersion, '9.4.0')) {
+                MainWindow.sendToRenderer(LOAD_INCOMPATIBLE_SERVER, this.id, loadURL.toString());
+                this.emit(LOAD_FAILED, this.id, 'Incompatible server version', loadURL.toString());
+                this.status = Status.ERROR;
+                return;
+            }
```

Comment:
Should this logic default to showing the error /unless/ we successfully get the remote info and confirm it's `>= 9.4.0`? Worried about edge cases with old servers that we won't know about vs. confidence in the server being new enough for this request and check to work reliably.

Reply:
Yeah that makes sense, the only thing I worry about is if we somehow don't get the configuration from the API and then they get the incompatible screen on a seemingly working server. I think though if we're somehow ending up not getting that info into the app that would be a different bug we'd want to expose, so let's go your way.

Reply:
If we default it to show, what happens if the user is on a spotty connection, is there a chance the version check fails and returns the error screen locking the user out? @devinbinnie 

Also for QA, I propose we spot check this on a couple newer release versions to ensure we aren't getting a false positive and displaying the error screen, especially on the v9.5 and v9.11 ESRs cc// @yasserfaraazkhan

Reply:
It would be a very small chance, but easily verifiable via refreshing. I think Jesse's point might cover more cases since it's clear we might have users on much much older versions.

Reply:
For example: 
<img width="621" alt="image" src="https://github.com/user-attachments/assets/16164985-0ce8-4196-8657-6f6b9c525d06" />

Reply:
Other than a spotty connection are there other cases where we might show this screen incorrectly sending the user down the wrong path (downgrading) to resolution ? 

For example:
- Server is unreachable momentarily
- Server is down for maintenance
etc

Worried we might be showing this screen in way too many cases that result in the user spending energy on unhelpful resolution steps that will make them more frustrated

Reply:
So to clarify, the only case where we'd be showing this screen is if with the two requests made to the server that check for version and just hit the homepage, if the version check one fails but the homepage one succeeds and is able to load the app. They should happen within milliseconds of each other.

The app makes somewhere around 50 requests to load the app, and if any one of them were to fail, the app would be in a degraded state. I think we might be trying to cover too many potential cases here, and not just focusing on the main issue of the server upgrade issue.

Remember that this needs to go into a patch release, so we want to minimize the amount of checks needed here. I think what we have is more than sufficient and will be plenty robust for all users.

Reply:
> So to clarify, the only case where we'd be showing this screen is if with the two requests made to the server that check for version and just hit the homepage, if the version check one fails but the homepage one succeeds and is able to load the app.

Got it, thanks